--- configure.in.orig	2010-06-11 01:14:46.000000000 +0300
+++ configure.in	2010-06-22 20:43:24.000000000 +0300
@@ -119,10 +119,10 @@
 
 fi
 
-AC_CHECK_LIB([fltk], [main], [ac_fltk="yes"], [ac_fltk="no"])
-if test "x$ac_fltk" = "xno"; then
-AC_MSG_ERROR([FLTK library not found.])
-fi
+#AC_CHECK_LIB([fltk], [main], [ac_fltk="yes"], [ac_fltk="no"])
+#if test "x$ac_fltk" = "xno"; then
+#AC_MSG_ERROR([FLTK library not found.])
+#fi
 
 AC_CHECK_LIB([Xpm],[main], [ac_Xpm="yes"], [ac_Xpm="no"])
 if test "x$ac_Xpm" = "xno"; then
